src/handlers/v1/markets/[chainId]/[marketId]/collateral-swap/index.ts

```diff
@@ -119,6 +119,7 @@ export const v1GetCollateralSwapQuotationRoute: Route<GetCollateralSwapQuotation
       approvals = estimateResult.approvals;
 
       // 6. calc target position
+      const targetSupplyUSD = new BigNumberJS(supplyUSD);
       const withdrawalUSD = new BigNumberJS(amount).times(withdrawalCollateral.assetPrice);
       const targetUSD = new BigNumberJS(quotation.output.amount).times(targetCollateral.assetPrice);
```

Comment:
```suggestion
      const withdrawalUSD = new BigNumberJS(amount).times(withdrawalCollateral.assetPrice);
      const targetUSD = new BigNumberJS(targetTokenAmount).times(targetCollateral.assetPrice);
      const targetSupplyUSD = new BigNumberJS(supplyUSD);
```
1. Follow the same order as other files
2. Use the existing `targetTokenAmount` for calculations